tests/NuGet.Services.AzureSearch.FunctionalTests/Support/NuGetSearchFunctionalTestBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BasicSearchTests.FunctionalTests.Core;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using Xunit;
+
+namespace NuGet.Services.AzureSearch.FunctionalTests
+{
+    /// <summary>
+    /// Base for functional tests on the NuGet search service.
+    /// </summary>
+    public class NuGetSearchFunctionalTestBase : BaseFunctionalTests, IClassFixture<CommonFixture>
+    {
+        public NuGetSearchFunctionalTestBase(CommonFixture fixture)
+            : base(fixture.TestSettings.AzureSearchAppServiceUrl)
+        {
+            Fixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+        }
+
+        protected CommonFixture Fixture { get; private set; }
+
+        /// <summary>
+        /// Queries the NuGet Search API.
+        /// See: https://docs.microsoft.com/en-us/nuget/api/search-query-service-resource#search-for-packages
+        /// </summary>
+        /// <param name="query">The search terms to filter packages.</param>
+        /// <returns>The package ids' that matches the query, lowercased.</returns>
+        protected async Task<IReadOnlyList<string>> SearchAsync(string query)
+        {
+            var requestUri = $"/query?q={HttpUtility.UrlEncode(query)}";
```

Comment:
nit: `semVerLevel=2.0.0` as this is the mainline scenario now

Consider including `prerelease=true` also since this the default on nuget.org

Reply:
Argh. I've made this mistake a bajillion times, you would think I would've learned by now.

Reply:
Fixed